test/inc/TestDef.h

```diff
@@ -23,6 +23,9 @@ static const std::wstring c_testRequestRestartNowRestartedPhasePackagedEventName
 
 static const std::wstring c_testPushPhaseEventName = L"WindowsAppRuntimeTestPushPhaseEventName";
 inline const winrt::hstring c_rawNotificationPayload = L"<toast></toast>";
+static const std::chrono::seconds c_timeout = std::chrono::seconds(300);
```

Comment:
Seems like a lot of constants specific to PushNotifications and not across the project. Better served in a PushNotifications directory, or at a minimum a filename less generic e.g.`PushNotifications-Test-Constants.h`